.circleci/config.yml

```diff
@@ -58,15 +58,15 @@ workflows:
           matrix:
             alias: old-ruby
             parameters:
-              ruby: ["2.7.8", "3.0.6", "3.1.4"]
+              ruby: ["2.7.8", "3.0.7", "3.1.6", "3.2.6", "3.3.7"]
```

Comment:
Added some other past versions, and set every one to the last patch version.

Reply:
I wonder if we still need to be testing against 2.7? But I guess it doesn't hurt.

Reply:
Probably we can get rid of it 👍

Reply:
looks like the tests on 2.7 failed anyway, so removing it is probably the easiest way forward there.